### PR TITLE
[kube-up][scale-out]Automatically create default network object (flat or mizar) of for system tenant

### PR DESCRIPTION
**Pull request by q131172019**
**What type of PR is this?**
/kind bug
/kind feature

**What this PR does / why we need it**:
The PR is to ensure to automatically create the network object (flat or mizar) of "default"  for system tenant when scale out 1 x 1 cluster or scale-up cluster of KUBE-UP environment.

If network support is not desired, setting env var DISABLE_NETWORK_SERVICE_SUPPORT will disable this feature.
 

**Which issue(s) this PR fixes**:
Fixes # 1279

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
TEST:
1. For mizar to start kube-up cluster:
start cluster:
```
export KUBEMARK_NUM_NODES=100 NUM_NODES=1 SCALEOUT_CLUSTER=true SCALEOUT_TP_COUNT=1 SCALEOUT_RP_COUNT=1 RUN_PREFIX=carltenant-012222 NETWORK_PROVIDER=mizar
```
```
export MASTER_DISK_SIZE=500GB MASTER_ROOT_DISK_SIZE=500GB KUBE_GCE_ZONE=us-west2-b MASTER_SIZE=n1-highmem-32 NODE_SIZE=n1-highmem-16 NODE_DISK_SIZE=500GB GOPATH=$HOME/go KUBE_GCE_ENABLE_IP_ALIASES=true KUBE_GCE_PRIVATE_CLUSTER=true CREATE_CUSTOM_NETWORK=true KUBE_GCE_INSTANCE_PREFIX=${RUN_PREFIX} KUBE_GCE_NETWORK=${RUN_PREFIX} ENABLE_KCM_LEADER_ELECT=false ENABLE_SCHEDULER_LEADER_ELECT=false ETCD_QUOTA_BACKEND_BYTES=8589934592 SHARE_PARTITIONSERVER=false LOGROTATE_FILES_MAX_COUNT=200 LOGROTATE_MAX_SIZE=200M KUBE_ENABLE_APISERVER_INSECURE_PORT=true KUBE_ENABLE_PROMETHEUS_DEBUG=true KUBE_ENABLE_PPROF_DEBUG=true TEST_CLUSTER_LOG_LEVEL=--v=2 HOLLOW_KUBELET_TEST_LOG_LEVEL=--v=2 GCE_REGION=us-west2-b
```
```
make quick-release; ./cluster/kube-up.sh
```
```
kubectl get network
```
```
NAME      TYPE    VPC                      PHASE   DNS
default   mizar   system-default-network 
```

Shutdown cluster:
```
./cluster/kube-down.sh
```
2. For flat network to start kube-up cluster:
start cluster:
```
export KUBEMARK_NUM_NODES=100 NUM_NODES=1 SCALEOUT_CLUSTER=true SCALEOUT_TP_COUNT=1 SCALEOUT_RP_COUNT=1 RUN_PREFIX=carltenant-012222
```
```
export MASTER_DISK_SIZE=500GB MASTER_ROOT_DISK_SIZE=500GB KUBE_GCE_ZONE=us-west2-b MASTER_SIZE=n1-highmem-32 NODE_SIZE=n1-highmem-16 NODE_DISK_SIZE=500GB GOPATH=$HOME/go KUBE_GCE_ENABLE_IP_ALIASES=true KUBE_GCE_PRIVATE_CLUSTER=true CREATE_CUSTOM_NETWORK=true KUBE_GCE_INSTANCE_PREFIX=${RUN_PREFIX} KUBE_GCE_NETWORK=${RUN_PREFIX} ENABLE_KCM_LEADER_ELECT=false ENABLE_SCHEDULER_LEADER_ELECT=false ETCD_QUOTA_BACKEND_BYTES=8589934592 SHARE_PARTITIONSERVER=false LOGROTATE_FILES_MAX_COUNT=200 LOGROTATE_MAX_SIZE=200M KUBE_ENABLE_APISERVER_INSECURE_PORT=true KUBE_ENABLE_PROMETHEUS_DEBUG=true KUBE_ENABLE_PPROF_DEBUG=true TEST_CLUSTER_LOG_LEVEL=--v=2 HOLLOW_KUBELET_TEST_LOG_LEVEL=--v=2 GCE_REGION=us-west2-b
```
```
make quick-release; ./cluster/kube-up.sh
```
```
kubectl get network
```
```
NAME      TYPE    VPC                      PHASE   DNS
default   flat 
```

Shutdown cluster:
```
./cluster/kube-down.sh
```

### CODE REVIEW

File: cluster/gce/gci/configure-helper-common.sh [hunk 1]
```diff
@@ -3240,6 +3225,10 @@ function start-mizar-scaleout {
   if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "tp" ]]; then
     create-mizar-operator-manifest
   fi
+  #TODO BUGBUG - REMOVE THIS BELOW, RP mizar-operator is only for testing
+  if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "rp" ]]; then
+    create-mizar-operator-manifest
```
[the remark this answers is not in the document]
Comment (vinaykul) [reply cluster/gce/gci/configure-helper-common.sh:3230]: Need to enable kube-proxy on tp.

File: cluster/gce/gci/configure.sh [hunk 2]
```diff
@@ -381,12 +398,8 @@ function install-cni-network {
   mkdir -p /etc/cni/net.d
   case "${NETWORK_POLICY_PROVIDER:-flannel}" in
     mizar)
-    if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
-      setup-mizar-cni-conf
-      install-mizar-cni-bin
-    else
-      download-mizar-cni-yaml
-    fi
+    setup-mizar-cni-conf
```
Comment (vinaykul) [inline cluster/gce/gci/configure.sh:401]: This might break scaleup.

File: cluster/gce/gci/configure.sh [hunk 3]
```diff
@@ -728,27 +732,6 @@ function install-kube-binary-config {
   rm -f "${KUBE_HOME}/${server_binary_tar}.sha1"
 }
 
-function ensure-mizar-kernel-and-ifname() {
```
Comment (vinaykul) [inline cluster/gce/gci/configure.sh:731]: We should keep this option for now. It is non-default but if we run into any blocking problems in ubuntu 20.04 we can fall back to this for debugging or verification.

**Comment by q131172019** [comment]
Close this PR and use PR1307 instead.

**Pull request closed without merging**